### ISSUE TITLE
3.40.0.4 breaks release build

### ISSUE THREAD

**Issue by bitbltr** [unoplatform/Uno.sqlite-wasm#33]
Download SQLiteSample (https://github.com/unoplatform/Uno.Samples/tree/master/UI/SQLiteSample)

Update Uno.sqlite-wasm:
`<PackageVersion Include="Uno.sqlite-wasm" Version="3.40.0.4" />`

Perform release build:
`msbuild SQLiteSample.csproj /r`

Receive error:
```
"E:\GitHub\unoplatform\Uno.Samples\UI\SQLiteSample\src\SQLiteSample\SQLiteSample.csproj" (default target) (1:31) ->
"E:\GitHub\unoplatform\Uno.Samples\UI\SQLiteSample\src\SQLiteSample\SQLiteSample.csproj" (Build target) (1:41) ->
(BuildDist target) ->
  C:\Users\---\.nuget\packages\uno.wasm.bootstrap\8.0.20\build\Uno.Wasm.Bootstrap.targets(226,3): error : System.
Exception: Failed to run emscripten. More details are available in diagnostics logging mode or using the MSBuild /bl sw
itch, see https://aka.platform.uno/msbuild-troubleshoot. [E:\GitHub\unoplatform\Uno.Samples\UI\SQLiteSample\src\SQLiteS
ample\SQLiteSample.csproj::TargetFramework=net8.0-browserwasm]
C:\Users\---\.nuget\packages\uno.wasm.bootstrap\8.0.20\build\Uno.Wasm.Bootstrap.targets(226,3): error :    at Uno
.Wasm.Bootstrap.ShellTask_vc1b86ef8ae35f8da1ee1065cb181faf0ad64afb5.RunPackager() in /_/src/Uno.Wasm.Bootstrap/ShellTas
k.cs:line 768 [E:\GitHub\unoplatform\Uno.Samples\UI\SQLiteSample\src\SQLiteSample\SQLiteSample.csproj::TargetFramework=
net8.0-browserwasm]
C:\Users\---\.nuget\packages\uno.wasm.bootstrap\8.0.20\build\Uno.Wasm.Bootstrap.targets(226,3): error :    at Uno
.Wasm.Bootstrap.ShellTask_vc1b86ef8ae35f8da1ee1065cb181faf0ad64afb5.Execute() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs
:line 271 [E:\GitHub\unoplatform\Uno.Samples\UI\SQLiteSample\src\SQLiteSample\SQLiteSample.csproj::TargetFramework=net8
.0-browserwasm]
```

All other packages can be updated without error:
```
<PackageVersion Include="SQLitePCLRaw.bundle_sqlite3" Version="2.1.10" />
<PackageVersion Include="SQLitePCLRaw.provider.sqlite3" Version="2.1.10" />
<PackageVersion Include="Uno.SQLitePCLRaw.provider.wasm" Version="3.2.0" />
<PackageVersion Include="SQLitePCLRaw.bundle_green" Version="2.1.10" />
```

**Comment by jeromelaban** [comment]
Thanks for the report. Indeed, this package was breaking older emscripten versions. 3.40.0.5 should fix it.

**Comment by bitbltr** [comment]
Confirmed fixed in 3.40.0.5. Thanks for the quick response!
